manim/cli/cfg/commands.py

```diff
@@ -1,23 +1,26 @@
-"""
-cfg_subcmd.py
-------------
+"""Manim's cfg subcommand.
 
-General Config File Managing Utilities.
-The functions below can be called via the `manim cfg` subcommand.
+Manim's cfg subcommand is accessed in the command-line interface via ``manim
+cfg``. Here you can specify options, subcommands, and subgroups for the cfg
+group.
 
 """
-
 import os
+import click
+
+from ...constants import EPILOG
+from ...constants import CONTEXT_SETTINGS
+
 from ast import literal_eval
 from typing import Union
 
-from manim import config, console
-from manim._config.utils import config_file_paths, make_config_parser
-from manim.utils.file_ops import guarantee_existence, open_file
+from .. import config, console
```

Comment:
sorry typo in suggestion previously
```suggestion
from ... import config, console
```

Reply:
Yeah, I fixed that in my recent commit. Also made the change in `plugins/commands.py`